src/lang.ts

```diff
@@ -160,6 +132,8 @@ export function getPropertyNames(object: {}): string[] {
 	return names;
 }
 
+// TODO ObjectObserver has a similar method that goes a step further by providing a default descriptor.
+// Should this implementation be replaced with that one, as this one is not used anymore?
```

Comment:
#40 discusses moving the observe stuff to its own file anyway, so does this become entirely removable then?  If so, I'd vote to remove.